st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
```

Comment:
I see, thanks for the clarification. I'm fine with that, I just found the name a bit confusing - alias for an alias! lol.

Anyways, LGTM, +1